com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/TestModule.java

```diff
@@ -51,9 +55,13 @@ protected void configure() {
     bind(LanguageServer.class).to(TestLanguageServer.class);
     bind(LanguageEngineFacade.class).to(CobolLanguageEngineFacade.class);
     bind(WorkspaceService.class).to(MockWorkspaceService.class);
+
+    bind(FileSystemService.class).to(FileSystemServiceImpl.class);
```

Comment:
Should it be MockFileSystemService here?

Reply:
I think is the same as https://github.com/eclipse/che-che4z-lsp-for-cobol/pull/172#discussion_r374076539. The new implementation will also involve this fix